test/org/sagebionetworks/bridge/dynamodb/DynamoSchedulePlanTest.java

```diff
@@ -0,0 +1,48 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import static org.junit.Assert.assertEquals;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Test;
+import org.sagebionetworks.bridge.json.BridgeObjectMapper;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class DynamoSchedulePlanTest {
```

Comment:
Nope, you can add it. Had to change some things to make the class final, and ignore the fact that the fields are mutable, but then it can be tested (and it found version wasn't included in hashCode/equals... pretty cool).